apis/r/tests/testthat/test-SOMAArrayReader-Iterated.R

```diff
@@ -152,8 +152,8 @@ test_that("Iterated Interface from SOMA Classes", {
 
 test_that("Iterated Interface from SOMA Sparse Matrix", {
     skip_if(!extended_tests() || covr_tests())
-    skip_on_ci()
     skip_if_not_installed("pbmc3k.tiledb")      # a Suggests: pre-package 3k PBMC data
+    skip_if(Sys.getenv("CI", "") != "")         # breaks only in CI so skipping
```

Comment:
Does this segfault in CI? There seems to be issues with `SOMAArray` segfaulting recently and just want to make note.

Reply:
Yes. This one is driving me mad. Cannot replicate elsewhere (tried several machines and setups) but fails at CI. It already had 'skip this' which I erreneously removed (as it passes locally here for me, and in other places).

At CI, at least for now, we do need it.  Once I add it back in, it all turns :heavy_check_mark: on all runs as you saw.